packages/bsp/common/usr/bin/armbianmonitor

```diff
@@ -146,24 +146,45 @@ ParseOptions() {
 			exit 0
 			;;
 		u)
-			# Upload /var/log/armbian-hardware-monitor.log with additional support info
-			fping ix.io 2>/dev/null | grep -q alive
-			if [ $? != 0 ]; then
-				echo -e "\nNetwork/firewall problem detected. Not able to upload debug info.\nPlease fix this or use \"-U\" instead and upload ${BOLD}whole output${NC} manually\n" >&2
-				exit 1
-			fi
-			which curl >/dev/null 2>&1 || apt-get -f -qq -y install curl
-			echo -e "System diagnosis information will now be uploaded to \c"
-			# we obfuscate IPv4 addresses somehow but not too much, MAC addresses have to remain
-			# in clear since otherwise the log becomes worthless due to randomly generated
-			# addresses here and there that might conflict
-			CollectSupportInfo \
-				| sed -E 's/([0-9]{1,3}\.)([0-9]{1,3}\.)([0-9]{1,3}\.)([0-9]{1,3})/XXX.XXX.\3\4/g' \
-				| curl -F 'f:1=<-' ix.io
-			echo -e "Please post the URL in the forum where you've been asked for.\n"
-			exit 0
-			;;
-		U)
+                        # Upload /var/log/armbian-hardware-monitor.log with additional support info
+                        which curl >/dev/null 2>&1 || apt-get -f -qq -y install curl
+                        echo -e "System diagnosis information will now be uploaded to \c"
+                        fping ix.io 2>/dev/null | grep -q alive
+                        if [ $? != 0 ]; then
+                                echo -e "\nNetwork/firewall problem detected.\nTrying fallback..." >&2
+                                fping paste.armbian.de 2>/dev/null | grep -q alive
+                                if [ $? != 0 ]; then
+                                        echo -e "\nNetwork/firewall problem detected. Not able to upload debug info.\nPlease fix this or use \"-U\" instead and upload ${BOLD}whole output${NC} manually\n" >>
+                                        exit 1
+                                fi
+
+                                # My bash foo isnt good enough to remove that function and use simple curl thingy like for ix.io
+                                fallback() {
+                                    local L="$1"
+                                    curl -X POST -s --data-binary @- "https://paste.armbian.de/documents" \
+                                    | awk -F '"' 'b{ b="."b }; {print a$4b}' a="https://paste.armbian.de/" b="${L}"
+                                }
+                                # we obfuscate IPv4 addresses somehow but not too much, MAC addresses have to remain
+                                # in clear since otherwise the log becomes worthless due to randomly generated
+                                # addresses here and there that might conflict
+                                CollectSupportInfo \
+                                        | sed -E 's/([0-9]{1,3}\.)([0-9]{1,3}\.)([0-9]{1,3}\.)([0-9]{1,3})/XXX.XXX.\3\4/g' \
+                                        | fallback
```

Comment:
```suggestion
                                # we obfuscate IPv4 addresses somehow but not too much, MAC addresses have to remain
                                # in clear since otherwise the log becomes worthless due to randomly generated
                                # addresses here and there that might conflict
                                CollectSupportInfo \
                                        | sed -E 's/([0-9]{1,3}\.)([0-9]{1,3}\.)([0-9]{1,3}\.)([0-9]{1,3})/XXX.XXX.\3\4/g' \
                                        | curl -s --data-binary @- "https://paste.armbian.de/documents" \
                                        | awk -F'"' '{ print "https://paste.armbian.de/" $4 }'
```